development_scripts/deconvolution_demo.py

```diff
@@ -0,0 +1,143 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Wed Jun  5 14:43:48 2024
+
+Script demonstrating the mass transport deconvolution class & methods
+
+email a.winiwarter@imperial.ac.uk for the data
+
+@author: awiniwar
+"""
+
+import matplotlib.pyplot as plt
+import numpy as np
+import ixdat
+
+from pathlib import Path
+from ixdat.techniques.deconvolution import ECMSImpulseResponse
+from ixdat import Measurement
+from ixdat.techniques.ms import MSCalResult
+from spectro_inlets_quantification import Calibration
+
+ixdat.config.plugins.activate_siq()
```

Comment:
I've been doing:
```
from ixdat import plugins

plugins.activate_siq()
```
Both do the same thing, but it'd be best to agree on just one "right" way to do it.

Reply:
I'm happy with either. I think from what you suggest may be clearer what it does. Will change  it to that and try and remember to use this from now on